scripts/b64.js

```diff
@@ -1,127 +0,0 @@
- * Nick Callen, Steven Levithan (stevenlevithan.com), Aidan Lister
- * (http://aidanlister.com/), Philippe Jausions
- * (http://pear.php.net/user/jausions), Zahlii, Denny Wardhana, Oskar Larsson
- * Högfeldt (http://oskar-lh.name/), Brian Tafoya
- * (http://www.premasolutions.com/), johnrembo, Gilbert, duncan, Thiago Mata
- * (http://thiagomata.blog.com), Alexander Ermolaev
- * (http://snippets.dzone.com/user/AlexanderErmolaev), Linuxworld, lmeyrick
- * (https://sourceforge.net/projects/bcmath-js/this.), Jon Hohle, Pyerre,
- * merabi, Saulo Vallory, HKM, ChaosNo1, djmix, Lincoln Ramsay, Adam Wallner
- * (http://web2.bitbaro.hu/), paulo kuong, jmweb, Orlando, kilops, dptr1988,
- * DxGx, Pedro Tainha (http://www.pedrotainha.com), Bayron Guevara, Le Torbi,
- * James, Douglas Crockford (http://javascript.crockford.com), Devan
- * Penner-Woelk, Jay Klehr, Kheang Hok Chin (http://www.distantia.ca/), Luke
- * Smith (http://lucassmith.name), Rival, Amir Habibi
- * (http://www.residence-mixte.com/), Blues (http://tech.bluesmoon.info/), Ben
- * Bryan, booeyOH, Dreamer, Cagri Ekin, Diogo Resende, Howard Yeend, Pul,
- * 3D-GRAF, jakes, Yannoo, Luke Godfrey, daniel airton wermann
- * (http://wermann.com.br), Allan Jensen (http://www.winternet.no), Benjamin
- * Lupton, davook, Atli Þór, Maximusya, Leslie Hoare, Bug?, setcookie, YUI
- * Library: http://developer.yahoo.com/yui/docs/YAHOO.util.DateLocale.html,
- * Blues at http://hacks.bluesmoon.info/strftime/strftime.js, Andreas,
- * Michael, Christian Doebler, Gabriel Paderni, Marco van Oort, Philipp
- * Lenssen, Arnout Kazemier (http://www.3rd-Eden.com), penutbutterjelly, Anton
- * Ongson, DtTvB (http://dt.in.th/2008-09-16.string-length-in-bytes.html),
- * meo, Greenseed, Yen-Wei Liu, mk.keck, William, rem, Jamie Beck
- * (http://www.terabit.ca/), Russell Walker (http://www.nbill.co.uk/),
- * Garagoth, Dino, Andrej Pavlovic, gabriel paderni, FGFEmperor, Scott Cariss,
- * Slawomir Kaniecki, ReverseSyntax, Mateusz "loonquawl" Zalega, Francois,
- * Kirk Strobeck, Billy, vlado houba, Jalal Berrami, date, Itsacon
- * (http://www.itsacon.net/), Martin Pool, Pierre-Luc Paour, ger, john
- * (http://www.jd-tech.net), mktime, Simon Willison
- * (http://simonwillison.net), Nick Kolosov (http://sammy.ru), marc andreu,
- * Arno, Nathan, Kristof Coomans (SCK-CEN Belgian Nucleair Research Centre),
- * Fox, nobbler, stensi, Matteo, Riddler (http://www.frontierwebdev.com/),
- * Tomasz Wesolowski, T.J. Leahy, rezna, Eric Nagel, Alexander M Beedie, baris
- * ozdil, Greg Frazier, Bobby Drake, Ryan W Tenney (http://ryan.10e.us), Tod
- * Gentille, Rafał Kukawski, FremyCompany, Manish, Cord, fearphage
- * (http://http/my.opera.com/fearphage/), Victor, Brant Messenger
- * (http://www.brantmessenger.com/), Matt Bradley, Luis Salazar
- * (http://www.freaky-media.com/), Tim de Koning, taith, Rick Waldron, Mick@el
- * 
- * Dual licensed under the MIT (MIT-LICENSE.txt)
- * and GPL (GPL-LICENSE.txt) licenses.
```

Comment:
Yeah, the new files each have their commentary, credits, and licensing intact. They're the base64_encode and utf8_encode files down lower in the diff.